### DISCUSSION TITLE
Hide panels on small window size

### DISCUSSION THREAD

**Discussion by fernandeusto** [rustdesk/rustdesk#4061] [Ideas]
### Description

Please add option to hide panels in rustdesk control UI

Option to hide left panel. and hide top panel
![image](https://user-images.githubusercontent.com/87156183/232174634-774d3b96-3e24-4b7d-a319-6e03ae14276f.png)

Someone like this:
![image](https://user-images.githubusercontent.com/87156183/232174743-fdcfafa7-0657-4925-8928-8a23940f2c76.png)

### Impact

it will let make small the window, and fast access to favorit clients

**Comment by rustdesk** [comment]
But PR is welcome.
